doc/api/child_process.markdown

```diff
@@ -735,10 +735,15 @@ If the process times out, or has a non-zero exit code, this method ***will***
 throw.  The `Error` object will contain the entire result from
 [`child_process.spawnSync`](#child_process_child_process_spawnsync_command_args_options)
 
+[EventEmitter]: events.html#events_class_events_eventemitter
+[net.Server]: net.html#net_class_net_server
+[net.Socket]: net.html#net_class_net_socket
```

Comment:
I'm not sure these should have moved. Isn't there a convention to put the link definitions at the end of the file ?